index.html

```diff
@@ -174,8 +174,40 @@ <h3>LET'S  CONNECT</h3>
                 <li><img src="./image/imgg1.png" alt="line2"></li>
             </ul>
             </div>
+            <div class="formy">
+                 <center> <div class="info-form">
+                    <h2 class="h002">Contact me</h2>
+                    <h3 class="h003">If you have an application you
+                         are interested in developing,
+                          a feature that you need
+                         built or a project that needs
+                          coding. I’d love to help with it</h3>
+
+                <form action="https://formspree.io/f/xyyvrpdv" method="POST">
+            <input class="input" type="text" name="name" placeholder="Enter your Name" maxlength="30">
+             <input class="input" type="email" placeholder="Enter your Email" name="email">
+             <input class="mssg" type="text" name="message" maxlength="500" placeholder="Enter your message here">                      
```

Comment:
- [ ] Well done on creating these input fields, however, there are a few requirements to add before we move into the next step, please consider adding the `required` attribute to all the input fields
- [ ] The message input field should be a `textarea` input instead, kindly modify it.